src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -139,9 +175,16 @@ function setup {
 	    copyToPFE "$PROJECT_DIR/$PROJECT_NAME"
 	    checkExitCode $? "Failed to copy project to PFE container."
     done
+
+    if [ $TEST_TYPE == "kube" ]; then
+        # Set up registry secrets (docker config) in PFE container/pod
+        setupRegistrySecret
+    fi
 }
 
 function run {
+    TURBINE_EXEC_TEST_CMD="cd /file-watcher/server; JUNIT_REPORT_PATH=/test_output.xml IMAGE_PUSH_REGISTRY_ADDRESS=${REGISTRY_SECRET_ADDRESS} IMAGE_PUSH_REGISTRY_NAMESPACE=${IMAGE_PUSH_REGISTRY_NAMESPACE} npm run $TEST_SUITE:test:xml"
```

Comment:
Any reason it was moved from the original place?

Reply:
since we're changing the address variable `REGISTRY_SECRET_ADDRESS` in function `setupInternalRegistryCredentials` above, we need the `TURBINE_EXEC_TEST_CMD` var to be declared after that function call since `TURBINE_EXEC_TEST_CMD` uses the address variable `REGISTRY_SECRET_ADDRESS`.

I can explain it better in person if that didnt make sense

Reply:
Nope, it makes sense now